sycl/test/regression/fsycl-save-temps.cpp

```diff
@@ -0,0 +1,23 @@
+//==--------------- fsycl-save-temps.cpp -----------------------------------==//
```

Comment:
I think, this test can find a better home somewhere at `clang/test`.

Reply:
From what I've seen before, it's not common to have full compilation runs within `clang/test`. I'll check again and gladly move the test upon success :)

Reply:
@alexbatashev I haven't found a good place for this one in `clang/test/*`. I'd appreciate a recommendation, or otherwise, would prefer to keep the current location.

Reply:
`clang/test/Driver`?
Anyway, I see we don’t have much e2e SYCL tests in clang/test, so, I’m ok to keep as is.

Reply:
Right, keeping it then. I've looked if similar Driver tests exist, but this doesn't seem to be the case.